UH-StudentServices/student_guide
# Feature/hub 318 user groups

# User groups for themes and instructions

- Added a new field: User group.
- Attached the field to Theme and Instruction (required).
- Refactored front page theme listing: Tab per user group.
- Degree programme specific news relocated. Displayed when degree students tab is active.

Testing:

- Import translations.
- Edit some Themes. Assign user groups.
- Verify that Themes fall under correct groups on front page.
